src/file_resolver.rs

```diff
@@ -39,25 +39,18 @@ pub struct ResolvedFile {
 
 impl FileResolver {
     /// Add import path
-    pub fn add_import_path(&mut self, path: &Path) -> io::Result<()> {
-        self.import_paths.push((None, path.canonicalize()?));
-        Ok(())
+    pub fn add_import_path(&mut self, path: &Path) {
+        self.import_paths.push((None, path.to_path_buf()));
```

Comment:
The struct fields doc comment says that `import_paths` is a set of path. Instead, it's just a list; what happens if duplicate paths are added? Does it matter? If yes, is it handled or prevented at an earlier stage?

Reply:
Good point. If you add the same path twice, import path resolution will detect that it can find the same file twice - via exactly the same path. That will be confusing, so we should detect this problem earlier and error about it.

Reply:
I've added code for this

Reply:
Nice 👍 Total nitpick: `import_paths` could be a HashSet then right?

Reply:
Also, fwiw, the simple fact that import path order doesn't matter is one of the reasons I like solang's interface so much more than solc's. Solc has --base-path and --include-path, and weird rules about when you can use each of them. But if you just use Solang's --import_path instead you get the exact same behavior.